db/schema.rb

```diff
@@ -128,6 +128,13 @@
     t.index ["poa_code"], name: "index_accredited_organizations_on_poa_code", unique: true
   end
 
+  create_table "accredited_representative_portal_verified_representatives", force: :cascade do |t|
```

Comment:
The indexes are missing in the schema

Reply:
Good eye! I forgot to commit it. Fixed:
https://github.com/department-of-veterans-affairs/vets-api/pull/16494/commits/856d17fca6c64e8d437a2d6e5f1339d04d84f3dd